code/controllers/subsystem/ticker.dm

```diff
@@ -397,8 +397,9 @@ SUBSYSTEM_DEF(ticker)
 
 /datum/controller/subsystem/ticker/proc/equip_characters()
 	var/captainless=1
-	for(var/mob/living/carbon/human/player in GLOB.player_list)
```

Comment:
player_list is pretty distinctively different from the human list. I'm not sure how good of an idea this is.

Reply:
I'll change these back!